pkg/network/node/healthcheck_ovs.go

```diff
@@ -0,0 +1,40 @@
+package node
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/openshift/sdn/pkg/util/ovsclient"
+	utilwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+)
+
+const (
+	ovsDialDefaultNetwork = "unix"
+	ovsDialDefaultAddress = "/var/run/openvswitch/db.sock"
+)
+
+func healthCheckOVS() error {
+	klog.Infof("[OVS healthcheck] starting OVS health check")
+	c, err := ovsclient.DialTimeout(ovsDialDefaultNetwork, ovsDialDefaultAddress, time.Minute)
+	if err != nil {
+		return fmt.Errorf("[OVS healthcheck] error connecting to OVS: %v", err)
```

Comment:
remove all the `[OVS healthcheck]` from the logs and errors.

for log messages, they will show in the logs as being from `healthcheck_ovs.go` anyway
for error messages, the message should assume it's going to get appended to some higher-level error message ("could not start up openshift-sdn: %v"). Having that `[]` part there just confuses things